external/kafka/src/test/scala/org/apache/spark/streaming/kafka/DirectKafkaStreamSuite.scala

```diff
@@ -381,3 +447,20 @@ object DirectKafkaStreamSuite {
     }
   }
 }
+
+private[streaming] class ConstantEstimator(rates: Double*) extends RateEstimator {
```

Comment:
Can you try to use the ConstantEstimator already defined in the Spark Streaming tests? 
https://github.com/apache/spark/blob/master/streaming/src/test/scala/org/apache/spark/streaming/scheduler/RateControllerSuite.scala#L89